itl/command-plugins-manubulon.conf

```diff
@@ -160,47 +187,61 @@ object CheckCommand "snmp-interface" {
 	command = [ ManubulonPluginDir + "/check_snmp_int.pl" ]
 
 	arguments += {
-		"-n" = "$snmp_interface$"
+		"-n" = {
+			value = "$snmp_interface$"
+			description = "Name in description OID (eth0, ppp0 ...). This is treated as a regexp : -n eth will match eth0,eth1,..."
+		}
 		"-k" = {
 			set_if = "$snmp_interface_perf$"
+			description = "Check the input/ouput bandwidth of the interface"
+		}
+		"--label" = {
+			value = "$snmp_interface_label$"
+			description = "Add label before speed in output : in=, out=, errors-out=, etc..."
 		}
-		"--label" = "$snmp_interface_label$"
 		"-Y" = {
 			set_if = "$snmp_interface_bits_bytes$"
+			description = "Output performance data in bits/s or Bytes/s"
 		}
 		"-y" = {
 			set_if = "$snmp_interface_percent$"
+			description = "Output performance data in % of max speed"
 		}
 		"-B" = {
 			set_if = "$snmp_interface_kbits$"
+			description = "Make the warning and critical levels in K|M|G Bits/s instead of K|M|G Bytes/s"
 		}
 		"-M" = {
 			set_if = "$snmp_interface_megabytes$"
+			description = "Make the warning and critical levels in Mbps"
 		}
 		"--64bits" = {
 			set_if = "$snmp_interface_64bit$"
+			description = "Use 64 bits counters instead of the standard counters when checking bandwidth & performance data for interface >= 1Gbps"
 		}
 		"-e" = {
 			set_if = "$snmp_interface_errors$"
+			description = "Add error & discard to Perfparse output"
 		}
 		"-i" = {
 			set_if = "$snmp_interface_inverse$"
+			description = "Make critical when up"
 		}
 		"-r" = {
 			set_if = "$snmp_interface_noregexp$"
+			description = "Do not use regexp to match NAME in description OID"
+		}
+		"-d" = {
+			value = "$snmp_interface_delta$"
+			description = "Make an average of <delta> seconds (default 300=5min)"
 		}
-		"-d" = "$snmp_interface_delta$"
 		"-u" = {
 			set_if = "$snmp_interface_warncrit_percent$"
-		}
-		"-N" = {
-			set_if = "$snmp_interface_ifname$"
-		}
-		"-A" = {
-			set_if = "$snmp_interface_ifalias$"
```

Comment:
Actually there is no parameter "-A" and "-N" in http://nagios.manubulon.com/snmp_int.html
That's why I removed it...